modules/tinymce/src/plugins/link/test/ts/browser/AssumeExternalTargetsTest.ts

```diff
@@ -1,114 +1,108 @@
-          ),
-          TestLinkUi.sClickConfirmYes,
-          TestLinkUi.sAssertContentPresence(tinyApis, { 'a': 1, 'a[href="http://www.google.com"]': 1 })
-        ])),
-
-        Logger.t('other urls are prompted to add http:// prefix', GeneralSteps.sequence([
-          TestLinkUi.sInsertLink(tinyUi, 'google.com'),
-          TestLinkUi.sWaitForUi(
-            'wait for dialog',
-            'p:contains("The URL you entered seems to be an external link. Do you want to add the required http:// prefix?")'
-          ),
-          TestLinkUi.sClickConfirmYes,
-          TestLinkUi.sAssertContentPresence(tinyApis, { 'a': 1, 'a[href="http://google.com"]': 1 })
-        ])),
-
-        Logger.t('url not updated when prompt canceled', GeneralSteps.sequence([
-          TestLinkUi.sInsertLink(tinyUi, 'google.com'),
-          TestLinkUi.sWaitForUi(
-            'wait for dialog',
-            'p:contains("The URL you entered seems to be an external link. Do you want to add the required http:// prefix?")'
-          ),
-          TestLinkUi.sClickConfirmNo,
-          TestLinkUi.sAssertContentPresence(tinyApis, { 'a': 1, 'a[href="google.com"]': 1 })
-        ]))
-      ]),
-
-      Log.stepsAsStep('TBA', 'link_assume_external_targets: http', [
-        tinyApis.sSetSetting('link_assume_external_targets', 'http'),
-
-        Logger.t('add http:// prefix to www-urls', GeneralSteps.sequence([
-          TestLinkUi.sInsertLink(tinyUi, 'www.google.com'),
-          TestLinkUi.sAssertContentPresence(tinyApis, { 'a': 1, 'a[href="http://www.google.com"]': 1 })
-        ])),
-
-        Logger.t('add http:// prefix to other urls', GeneralSteps.sequence([
-          TestLinkUi.sInsertLink(tinyUi, 'google.com'),
-          TestLinkUi.sAssertContentPresence(tinyApis, { 'a': 1, 'a[href="http://google.com"]': 1 })
-        ]))
-      ]),
-
-      Log.stepsAsStep('TBA', 'link_assume_external_targets: https', [
-        tinyApis.sSetSetting('link_assume_external_targets', 'https'),
-
-        Logger.t('add https:// prefix to www-urls', GeneralSteps.sequence([
-          TestLinkUi.sInsertLink(tinyUi, 'www.google.com'),
-          TestLinkUi.sAssertContentPresence(tinyApis, { 'a': 1, 'a[href="https://www.google.com"]': 1 })
-        ])),
-
-        Logger.t('add https:// prefix to other urls', GeneralSteps.sequence([
-          TestLinkUi.sInsertLink(tinyUi, 'google.com'),
-          TestLinkUi.sAssertContentPresence(tinyApis, { 'a': 1, 'a[href="https://google.com"]': 1 })
-        ]))
-      ])
-    ], onSuccess, onFailure);
-  }, {
+describe('browser.tinymce.plugins.link.AssumeExternalTargetsTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
     plugins: 'link',
     toolbar: 'link',
-    theme: 'silver',
     base_url: '/project/tinymce/js/tinymce'
-  }, success, failure);
+  }, [ Plugin, Theme ]);
+
+  context('TBA: Default setting', () => {
```

Comment:
There probably shouldn't be a `TBA` here, as that's meant to be on individual tests, not on a context/describe.